### ISSUE TITLE
Current Limitation by FilterBlast_api;

### ISSUE THREAD

**Issue by zombodotcom** [POE-Addon-Launcher/PoE-Addon-Launcher#22]
It's a good and quick solution but in the future, the only way that updates like the multiple colors folders in 
[NeverSinks Github](https://github.com/NeverSinkDev/NeverSink-Filter) Wont ever be added unless FilterBlade updates their API to include those. 

just a heads up

**Comment by POE-Addon-Launcher** [comment]
Yep, I'm currently working on something for this and hoping to release it soon.
